### ISSUE TITLE
Home Page>Tutor View>Student Overview add Course Title

### ISSUE THREAD

**Issue by michaelwebbjisc** [jiscdev/data-explorer#120]
From USW

**Comment by leebaylis** [comment]
Freshdesk ticket was left open so Neil Price commented there:

"We are seeing Not Known now in Course. It the no link to the student and the course instance?"

**Comment by leebaylis** [comment]
Freshdesk ticket: 
https://jisclearninganalytics.freshdesk.com/helpdesk/tickets/235

**Comment by pdiveris** [comment]
Looking the data I've got from USW the tutees response doesn't contain any studentCourseMemberships. If it did then the code would most likely produce a string in the form of "58c079d6321c95ab45c0903a" which I am not sure exactly how useful is for the end user. The tutees response does contain a populated studentCourseInstances with a similar kind of code, so if that's of any use then I could just display that. I suspect that (as it's the case with the fuzzySearch) some work needs doing on the query side itself so that the calls return more useful results.

**Comment by leebaylis** [comment]
Yes, if the query doesn't join the course table through the studentcoursemembership table it will not be useful data...

**Comment by robbristow-jisc** [comment]
Not sure what we are looking for here. Is this relevant now?

**Comment by michaelwebbjisc** [comment]
![image](https://user-images.githubusercontent.com/8233360/31613011-b37d45b0-b279-11e7-81b6-7ce88b515046.png)
